app/uk/gov/hmrc/uknwauthcheckerapistub/controllers/EisStubController.scala

```diff
@@ -26,12 +26,21 @@ import javax.inject.{Inject, Singleton}
 class EisStubController @Inject() (cc: ControllerComponents) extends BackendController(cc) with Helper {
 
   def authorisations(): Action[AnyContent] = Action { implicit request: Request[AnyContent] =>
+    val myHead = request.headers.headers
     val myBody = request.body.asJson
 
-    myBody.get.validate[Eoris] match {
-      case eoris => Ok(makeAJsonRes(eoris.get))
-      case _     => InternalServerError
+    val valid = myHead.filter(_._1.contains("authorization"))
```

Comment:
Can you move all this into the utils? Would remove a lot of the boilerplate from this method.

e.g. if (hasValidBearerToken(request)) {
} else {
   Forbidden
}